code/modules/library/skill_learning/job_skillchips/research_director.dm

```diff
@@ -1,3 +1,5 @@
+#define SKILLCHIP_CATEGORY_GENERAL "general"
+
```

Comment:
Two last things: 

1. This was already defined and undefined somewhere, so we can improve upon this. We should put both defines in some define file, such as `skills.dm`:

```dm
#define SKILLCHIP_CATEGORY_GENERAL "general"
#define SKILLCHIP_CATEGORY_JOB "job"
```

2. I think this deserves its own category, even if it fundamentally makes no difference, such as: 

```dm
#define SKILLCHIP_CATEGORY_JOKE "meme"
```
or
```dm
#define SKILLCHIP_CATEGORY_LOW_IMPACT "low_impact" 
```

to denote the skillchip as one which isn't extremely useful in average station activities

Reply:
> Two last things:
> 
> 1. This was already defined and undefined somewhere, so we can improve upon this. We should put both defines in some global define file, such as `skills.dm`:
> 
> ```dm
> #define SKILLCHIP_CATEGORY_GENERAL "general"
> #define SKILLCHIP_CATEGORY_JOB "job"
> ```
> 
> That way we don't have to undef them or re-re-define them
> 
> 2. I think this deserves its own category, even if it fundamentally makes no difference, such as:
> 
> ```dm
> #define SKILLCHIP_CATEGORY_JOKE "meme"
> ```
> 
> or
> 
> ```dm
> #define SKILLCHIP_CATEGORY_LOW_IMPACT "low_impact" 
> ```
> 
> to denote the skillchip as one which isn't extremely useful in average station activities

So i have to move more than RD's chip only into Meme category, because there are few more chips, that are too low impact then others

Reply:
Wine Skillchip especially

Reply:
A-a-and no, i have to re-define

Reply:
> Two last things:
> 
> 1. This was already defined and undefined somewhere, so we can improve upon this. We should put both defines in some global define file, such as `skills.dm`:
> 
> ```dm
> #define SKILLCHIP_CATEGORY_GENERAL "general"
> #define SKILLCHIP_CATEGORY_JOB "job"
> ```
> 
> That way we don't have to undef them or re-re-define them
> 
> 2. I think this deserves its own category, even if it fundamentally makes no difference, such as:
> 
> ```dm
> #define SKILLCHIP_CATEGORY_JOKE "meme"
> ```
> 
> or
> 
> ```dm
> #define SKILLCHIP_CATEGORY_LOW_IMPACT "low_impact" 
> ```
> 
> to denote the skillchip as one which isn't extremely useful in average station activities

Can you delete this change request? It's done and it can't be merged cuz request